docs/installation/virtual/proxmox.rst

```diff
@@ -7,31 +7,48 @@ Running on Proxmox
 Proxmox is an open-source platform for virtualization. Users with a support
 subscription can download a qcow2 image that can be imported into Proxmox.
 
-Deploy VyOS from CLI
+Deploy VyOS from CLI with qcow2 image
 ====================
```

Comment:
please adjust the length of the `=` it must have the same as the headline above.